apps/cookbook/src/app/examples/page-example/advanced/page-custom-title-example.component.ts

```diff
@@ -10,18 +10,27 @@ const config = {
   template: `<kirby-page defaultBackHref="/">
 
   <!-- Custom Page Title -->
-  <h1 *kirbyPageTitle [kirbyFitHeading]="{ maxLines: 2 }">
-    <ng-container *ngTemplateOutlet="customTitle"></ng-container>
-  </h1>
+  <div *kirbyPageTitle>
+    <ng-container *ngTemplateOutlet="pageTitle"></ng-container>
+  </div>
 
   <ng-template kirbyPageToolbarTitle>
-    <ng-container *ngTemplateOutlet="customTitle"></ng-container>
+    <ng-container *ngTemplateOutlet="toolbarTitle"></ng-container>
   </ng-template>
 
-  <ng-template #customTitle>
-    <div style="display: flex;">
-      <div style="overflow: hidden; text-overflow: ellipsis;">
-        Custom Title With a very long name
+  <ng-template #pageTitle>
+    <div style="display: flex; gap: 0.5rem;">
```

Comment:
Consider moving inlined styles to comp's stylesheet

Reply:
I don't think there is a stylesheet for this example component. The inline styles were already there (except for the gap I added).